Projects/App/Sources/UI/User/Profile/View/ProfileUserMenuView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ProfileUserMenuView.swift
+//  App
+//
+//  Created by Keum MinSeok on 2022/10/26.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+final class ProfileUserMenuView: UIView {
+    
+    // MARK: Properties
+        
+    private let userMenuLabel = UILabel()
+    
+    // MARK: LifeCycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureUI()
+        createLayout()
+    }
+    
+    convenience init(userMenuText: String) {
+        self.init(frame: .zero)
+        userMenuLabel.text = userMenuText
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+}
+
+extension ProfileUserMenuView {
+    
+    // MARK: UI Function
+    
+    private func configureUI() {
+        backgroundColor = .zestyColor(.background)
+
+        userMenuLabel.font = .systemFont(ofSize: 16, weight: .medium)
+        userMenuLabel.textColor = .black
```

Comment:
p2;

로그아웃과 회원탈퇴 버튼은 놀랍게도 색상이 있습니다.

```suggestion
        userMenuLabel.textColor = .zestyColor(.dim)
```

아 그리고 이 파일에 DesignSystem 임포트 안되어있어서 얘도 임포트 해줘야될듯요 (안해줘도 오류 안나는데 이유는 모르겠음)

Reply:
9b585061958facb82380af2c82f10f6d481c6f01 헉! 색상이 있는 줄 몰랐네요🫠